docs/src/main/asciidoc/infinispan-client.adoc

```diff
@@ -357,6 +424,27 @@ annotation is not required and if it is not supplied, the default cache will be
 
 NOTE: Other types may be supported for injection, please see other sections for more information
 
+=== Registering Protobuf Schemas with Infinispan Server
+
+One the serialization on the client guide is done, you must register Protobuf schemas
+with Infinispan Server if you want to perform queries or convert from `Protobuf` to other media types
+such as `json`.
```

Comment:
Should json be capitalized? I don't know :)

Reply:
i capitalize